hw3/01-characters.js

```diff
@@ -1,4 +1,69 @@
-// url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+// GET api/characters
+// [
+//     {
+//       "id": 0,
+//       "firstName": "string",
+//       "lastName": "string",
+//       "fullName": "string",
+//       "title": "string",
+//       "family": "string",
+//       "image": "string",
+//       "imageUrl": "string"
+//     }
+//   ]
```

Comment:
This comment is useful for development and debugging, but it does not necessarily need to be merged into main.